cogs/command_utils/games_util.py

```diff
@@ -0,0 +1,396 @@
+
+        self.react_controls = [
+            ("💎", self.rock),
+            ("📜", self.paper),
+            ("✂️", self.scissor),
+            ("\N{BLACK SQUARE FOR STOP}", self.stop),
+        ]
+
+        self.current_round = 0
+        self.current_player = 0
+
+        self.win_lose = [0, 0]  # index 0 is win | index 1 is lose
+        self.total_rounds = rounds
+        self.game_state = 0
+        self.game_text = {"starting": "starting...", "pick": "Pick a option...", "await": "Awaiting response from {}",
+                          "win": "{} you've won the round!", "lose": "{} lost the round!",
+                          "tie": "The round is a tie!", "win_game": "Congratulations you have won the game!",
+                          "lose_game": "Unfortunately you have lost the game",
+                          "tie_game": "Looks like the game was a tie!"}
+        self.meta_info = ""
+
+        self.players = self.order_players(players)
+        self.player_responses = {ply.user.mention: None for ply in self.players.values()}
+
+    def get_player(self):
+        self.current_player += 1
+        if self.current_player > len(self.players) - 1:
+            self.current_player = 0
+        return self.players[self.current_player]
+
+    def peek_player(self):
+        return self.players[self.current_player]
+
+    def order_players(self, players):
+        return {index: RPSPlayer(user, self.total_rounds) for index, user in enumerate(players)}
+
+    def reset_responses(self):
+        self.player_responses = {ply.user.mention: None for ply in self.players.values()}
+
+    def format_string(self, get, fmt=None):
+        if fmt:
+            return self.game_text[get].format(fmt)
+        else:
+            return self.game_text[get]
+
+    async def ensure_responses(self, response):
+        ply = self.peek_player()
+        self.player_responses[ply.user.mention] = response
+        if len([null for null in self.player_responses.values() if null is not None]) != len(self.players.values()):
+            return False
+        else:
+            return True
+
+    async def calculate_win(self):
+        def calc_win(first_player, second_player):
+            """Calculates winner
+            If resulting winner is player_one then returns 0
+            If resulting winnner is player_two returns 1
+            if tied returns 2
+            """
+            if (second_player % 3) + 1 == first_player:
+                return 0
+            elif (first_player % 3) + 1 == second_player:
+                return 1
+            else:
+                return 2
+
+        responses = self.player_responses.values()
+        plys = [key for key in self.player_responses.keys()]
+        self.reset_responses()
+
+        result = calc_win(*responses)
+        player_one_words, player_two_words = await self.rps_to_words(*responses)
+        if result is 0:
+            return True, f"{plys[0]}'s {player_one_words} beats {plys[1]}'s {player_two_words}"
```

Comment:
F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
R1705: Unnecessary "elif" after "return"